src/components/TodoInput.js

```diff
@@ -0,0 +1,17 @@
+export default function TodoInput({ onAdd }) {
+  const $todoInput = document.querySelector('#new-todo-title');
+
+  $todoInput.addEventListener('keydown', (event) => this.addTodoItem(event));
+
+  this.addTodoItem = (event) => {
+    if (event.key !== 'Enter') return;
+
+    const todoInputTarget = event.target;
+    if (todoInputTarget.value === '') return;
+
+    onAdd(todoInputTarget.value);
+    todoInputTarget.value = '';
+  };
+
+  $todoInput.focus();
```

Comment:
html문서에 autofocus를 사용하는 것에  비해 focus 이벤트를 사용했을 떄의 장점이 궁금합니다! ㅎㅎㅎ

Reply:
'해야할 일'이나 '완료한 일' 버튼을 클릭하면, 주소창에 #active나 #completed가 붙잖아요. 이 버튼을 누른 뒤, 새로고침하면 크롬 브라우저 콘솔 창에 `Autofocus processing was blocked because a document already has a focused element.` 라는 에러가 뜨면서 자동으로 포커스가 안되더라구요. 버그인 것 같은데, 명쾌한 해결법은 찾지 못해서 그냥 자바스크립트로 focus 하는 방식으로 했습니다!